lib/redis_memo/future.rb

```diff
@@ -37,15 +37,21 @@ def method_cache_key
       RedisMemo::MemoizeMethod.method_cache_keys([context])&.first || ''
   end
 
+  def validate_cache_result
+    cache_validation_sample_percentage = @cache_options[:cache_validation_sample_percentage]&.to_i
+    if cache_validation_sample_percentage.nil?
+      RedisMemo::DefaultOptions.cache_validation_sampler&.call(@method_id)
```

Comment:
Should this return false, indicating that we should not sample if no value is provided?

Reply:
if it returns false, that means no `cache_validation_sample_percentage` is not defined in the inline method, then it will fall back to the default global behavior (previous line 46 - 47)

Reply:
> Will have a next PR to replace the config.cache_validation_sampler with config.cache_validation_percentage

Got it, I missed this message too. This makes sense, but it's a bit confusing to me to make these changes separately. In the intermediate state, we'll either check the cache options for the percentage, or check the global cache validation sampler (which inherently would do the same thing by checking the percentage).

Reply:
Done. Added the in the `config.cache_validation_percentage` in new commit